### PR TITLE
feat(fuselage): Empty State Component

### PR DESCRIPTION
**Pull request by rique223**
- [x] I have read the Contributing Guide
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have labeled the PR correctly with the related package
- [x] I have run Loki's visual regression tests (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] I have added necessary documentation (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules

## Proposed changes (including videos or screenshots)
image

### CODE REVIEW

File: packages/fuselage/src/components/Fallback/Fallback.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import {
+  Fallback,
+  FallbackSubtitle,
+  FallbackIcon,
+  FallbackTitle,
+} from './Fallback';
+
+export default {
+  title: 'Fallback/Fallback',
+  //   component: Message,
```
Comment (ujorgeleite) [inline packages/fuselage/src/components/Fallback/Fallback.stories.tsx:21]: Can we remove this unused and commented code?

File: packages/fuselage/src/components/Fallback/Fallback.styles.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  &__suggestion {
+    @include typography.use-font-scale(p2);
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    width: 100%;
+    max-width: 276px;
```
Comment (ujorgeleite) [inline packages/fuselage/src/components/Fallback/Fallback.styles.scss:41]: This fixed number can be replaced with a dynamic value or inherit sizes?

File: packages/fuselage/src/components/Fallback/Fallback.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import { Icon } from '../Icon';
+import './Fallback.styles.scss';
+
+export const Fallback: FC = ({ children }) => (
+  <div className='rcx-fallback'>{children}</div>
+);
+
+export const FallbackIcon: FC<{
+  name: ComponentProps<typeof Icon>['name'];
+  // variation?: 'danger';
```
Comment (ujorgeleite) [inline packages/fuselage/src/components/Fallback/Fallback.tsx:14]: I suggest to remove this unused code.

**Review by ujorgeleite — CHANGES_REQUESTED** [review]

**Review by dougfabris — CHANGES_REQUESTED** [review]
I think we should remove the loki refs with the old component name

**Review by dougfabris — APPROVED** [review]

**Review by dougfabris — APPROVED** [review]

**Pull request merged**